runtime/near-runtime-fees/src/lib.rs

```diff
@@ -85,6 +85,18 @@ pub struct ActionCreationConfig {
     pub delete_account_cost: Fee,
 }
 
+/// Describes the cost of creating an access key.
+#[derive(Debug, Serialize, Deserialize, Clone, Hash, PartialEq, Eq)]
+pub struct AccessKeyCreationConfig {
+    /// Base cost of creating a full access access-key.
+    pub full_access_cost: Fee,
+    /// Base cost of creating an access-key restricted to specific functions.
+    pub function_call_cost: Fee,
+    /// Cost per byte (includes allowance, receiver_id, method_names) of creating a restrictec
```

Comment:
typo: restricted

I think we shouldn't account for receiver_id length (should be fixed cost), because it's limited to 64 chars and we use fixed cost for routing receipts.

Reply:
Fixed